cmd/cmrel/cmd/staged.go

```diff
@@ -139,3 +190,11 @@ func logTable(lines ...string) {
 	}
 	w.Flush()
 }
+
+type ByVersion []release.Staged
+
+func (a ByVersion) Len() int      { return len(a) }
+func (a ByVersion) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a ByVersion) Less(i, j int) bool {
+	return strings.Compare(a[i].Metadata().ReleaseVersion, a[j].Metadata().ReleaseVersion) < 0
+}
```

Comment:
I thought I added this comment in previous review.
I suggest we use https://pkg.go.dev/golang.org/x/mod/semver#Compare so that `v1.2.0` is correctly sorted before `v1.10.0`

Reply:
Fixed in d538ce9.